### PR TITLE
Update route describe decorator to use autoDescribeRoute

### PR DESCRIPTION
**Pull request by cjh1**
Relies on girder/girder#1701

### CODE REVIEW

File: server/hpccloud/plugin_tests/projects_test.py [hunk 1]
```diff
@@ -127,292 +127,292 @@ def test_create(self):
...
+#         r = self.request('/folder/%s' % str(project1['folderId']), method='GET',
+#                          type='application/json', user=self._another_user)
+#         self.assertStatus(r, 200)
+#         self.assertEqual(r.json['_id'], project1['folderId'])
+#
+#         # Check that owner still has access
+#         r = self.request('/projects/%s' % str(project1['_id']), method='GET',
+#                  type='application/json', user=self._yet_another_user)
+#         self.assertStatus(r, 200)
+#         self.assertEqual(r.json['_id'], project1['_id'])
```
Comment (TristanWright) [inline server/hpccloud/plugin_tests/projects_test.py]: did you mean to leave this all commented out?

**Comment by cjh1** [comment]
@TristanWright Hold off on review, looks like there is a whole test commented out!? not sure why ...

File: server/hpccloud/plugin_tests/projects_test.py (hunk 1 above)
Comment (cjh1) [inline server/hpccloud/plugin_tests/projects_test.py]: :smile Not sure looking into it! It was a while ago!

Comment (cjh1) [inline server/hpccloud/plugin_tests/projects_test.py]: Tests fail, I guess that is why they are commented out :smile:

**Comment by cjh1** [comment]
@TristanWright OK, fixed up.

**Pull request merged**